public/app/plugins/datasource/prometheus/components/monaco-query-field/monaco-completion-provider/index.ts

```diff
@@ -4,6 +4,30 @@ import { getSituation } from './situation';
 import { getCompletions, DataProvider, CompletionType } from './completions';
 import { NeverCaseError } from './util';
 
+export function getSuggestOptions(): monacoTypes.editor.ISuggestOptions {
+  return {
```

Comment:
Do we have any guidelines on the length of comment blocks? This one looks like a poem :D

Reply:
i could make it rhyme maybe? :-) 

i made them longer-and-less-lines . i tried to fit into 80characters/row.

Reply:
Looks good, but rhymed comments would be fantastic :D